reproman/distributions/conda.py

```diff
@@ -60,7 +61,7 @@ def get_conda_platform_from_python(py_platform):
     return None
 
 
-def get_miniconda_url(conda_platform, python_version):
+def get_miniconda_url(conda_platform, python_version, conda_version):
```

Comment:
adjust docstring to reflect adding `conda_version`. But overall I feel that ideally we should allow for it to be just `latest` if not specified. I don't think we would be able to find installations for all versions of `conda` -- people might install for some conda version and then upgrade it.  Since not directly needed ATM

Reply:
fixed